CAM Platform/api-cost-centers.md

```diff
@@ -0,0 +1,337 @@
+| deleted | object | Identifies whether the Cost Center is deleted. |
+| id | string | Cost Center unique identifier. |
+| members | array | Members' list of a Cost Center. |
+| name | string | Cost Center name |
+| organization | string | Organization which the Cost Center belongs to. |
+| schema | string | Schema URI. http://elasticbox.net/schemas/costcenter. |
+| updated | string | Date of the last update. Example “2018-10-30 16:03:14.409029” |
+| uri | string | Cost Center URI. |
+
+#### Response Body
+
+```
+{
+    "updated": "2018-10-30 16:03:14.409029",
+    "name": "Cost Center example",
+    "created": "2015-07-02 10:23:47.748740",
+    "deleted": null,
+    "uri": "/services/costcenters/5be2de28-1862a-3238-9a50d81",
+    "members": [
+        {
+            "type": "workspace",
+            "role": "administrator",
+            "id": "elasticboxdemo"
+        },
+    ],
+    "clc_alias": "A79A",
+    "organization": "elasticboxdemo",
+    "id": "5be2de28-1862a-3238-9a50d81",
+    "schema": "http://elasticbox.net/schemas/costcenter"
+}
+```
+
+## PUT /services/costcenters
+
+Updates one or several values on an existing Cost Center of the authenticated user.
+
+**Note:**
+Be careful using this command because you can add or remove some values depending on whether they are included or not as a request parameter. We recommend first obtain this data with GET command and then make changes over them. 
+
+### URL
+
+#### Structure
+```
+[PUT] /services/costcenters/{costcenter_id}
+```
+#### Example
+```
+[PUT] https://cam.ctl.io/services/costcenters/5be2de281862a32389a50d81
+```
+### Request
+
+#### Headers
+```
+Content-Type: application/json
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
+#### URI Parameters
+| Name | Type | Description | Req. |
+|------| ---- | ----------- | ---- |
+| costcenter_id | string | Cost Center ID | Yes |
+
+#### Request body parameters
+
+| Name | Type | Description | Req. |
+|------| ---- | ----------- | ---- |
+| id | string | Cost Center unique identifier. | Yes |
+| members | array | Members' list of the Cost Center. | Yes |
+| name | string | Cost Center name. | Yes |
+| organization | string | Organization which the Cost Center belongs to. | Yes |
+| schema | string | Schema URI. http://elasticbox.net/schemas/costcenter. | Yes |
+| account_installed_product | string | Account Installed Product. | No |
+| billing_account_number | string | Billing account number. | No |
+| icon_metadata | string | Icon used for the Cost Center account.. | No |
+
+**Note:**
+Be careful using this command because some values like *members* will be added or deleted if they exists or not in the request parameter 
+
+```
+{
+    "id":"dbfdd7a7-c348-4d55-817a-5890699bbf86",
+    "name":"Cost Center example",
+    "organization":"elasticboxdemo",
+    "schema":"http://elasticbox.net/schemas/costcenter",
+    "members":[{"id":"elasticboxdemo"}]
```

Comment:
Use a full cost center object, like the one in the GET example, to show that this should include all parameters returned by a GET.

Reply:
Parameters not shown must not be provided. They are not updated when requesting this command